src/python/pants/backend/go/goals/tailor.py

```diff
@@ -95,7 +95,7 @@ async def find_putative_go_targets(
         ]
         existing_targets = await Get(
             UnexpandedTargets,
-            Specs(ancestor_globs=tuple(AncestorGlobSpec(d) for d in main_package_dirs)),
+            RawSpecs(ancestor_globs=tuple(AncestorGlobSpec(d) for d in main_package_dirs)),
         )
```

Comment:
FYI: this uses `RawSpecs` because we're not operating on the CLI specs the user provided us. This is a hardcoded query.

We could also use `Specs` just fine, and set `ignores=RawSpecs()`! Only it's more boilerplate.

--

Ditto the other rules.

Reply:
Got it.